### DISCUSSION TITLE
How do I change the default storage directory to a different location?

### DISCUSSION THREAD

**Discussion by Livable8209** [containers/podman#24436] [Q&A]
Please bear in mind I'm a new user of podman, so I haven't had time to experiment.

I've spent some time looking around trying to figure out how to move to an external drive, but I haven't had any luck doing so.

I've freshly installed podman on Debian 12, and I've tried making storage.conf in `/etc/containers` and `$HOME/.local/share/containers/storage/`, neither of which have worked for me.

I used the following storage.conf for both of these attempts:

```toml
[storage]
driver="vfs"
graphroot="/mnt/extern/podman"
```

What i want to do is move all images and containers onto this external drive. Have I missed anything important?

(p.s, I've fully uninstalled podman from my server for now, and will instead experiment inside a vm)

**Comment by Anachron** [comment]
This is my `storage.conf`:

```
[storage]
driver = "overlay"
runroot = "/run/cnt"
graphroot = "/data/cnt/graph"
rootless_storage_path = "/data/cnt/rootless"
```

You can then run `podman info` and check if there are still some paths on your local drive in the `store` section.
